cpp/oneapi/dal/detail/policy.hpp

```diff
@@ -43,7 +43,10 @@ class ONEDAL_EXPORT default_host_policy {};
 
 class ONEDAL_EXPORT host_policy : public base {
 public:
-    host_policy();
+    static host_policy& instance() {
```

Comment:
According to our conventions

```suggestion
    static host_policy& get_instance() {
```

Reply:
Does `static const host_policy&` work?